catalog/pipeline/fbc-release/0.18/fbc-release.yaml

```diff
@@ -0,0 +1,343 @@
+    - name: requestReason
+      value: $(tasks.add-fbc-contribution-to-index-image.results.requestReason)
+  tasks:
+    - name: collect-data
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: https://github.com/redhat-appstudio/release-service-bundles.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: catalog/task/collect-data/0.1/collect-data.yaml
+      params:
+        - name: release
+          value: $(params.release)
+        - name: releaseplan
+          value: $(params.releaseplan)
+        - name: releaseplanadmission
+          value: $(params.releaseplanadmission)
+        - name: releasestrategy
+          value: $(params.releasestrategy)
+        - name: snapshot
+          value: $(params.snapshot)
+      workspaces:
+        - name: data
+          workspace: release-workspace
+    - name: verify-enterprise-contract
+      taskRef:
+        resolver: "bundles"
+        params:
+          - name: bundle
+            value: $(params.verify_ec_task_bundle)
+          - name: kind
+            value: task
+          - name: name
+            value: verify-enterprise-contract
+      params:
+        - name: IMAGES
+          value: "$(workspaces.data.path)/snapshot_spec.json"
+        - name: SSL_CERT_DIR
+          value: /var/run/secrets/kubernetes.io/serviceaccount
+        - name: POLICY_CONFIGURATION
+          value: $(params.enterpriseContractPolicy)
+        - name: STRICT
+          value: "1"
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - collect-data
+    - name: get-ocp-version
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: https://github.com/redhat-appstudio/release-service-bundles.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: catalog/task/get-ocp-version/0.1/get-ocp-version.yaml
+      params:
+        - name: fbcFragment
+          value: $(params.fbcFrgment)
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - verify-enterprise-contract
+    - name: update-version-tag
+      workspaces:
+        - name: input
+          workspace: release-workspace
+      runAfter:
+        - get-ocp-version
+      params:
+        - name: image
+          value: "quay.io/hacbs-release/release-utils:4d8649dbb2b626f5fe9f4ff83c1bc3be268fad31"
+        - name: script
+          value: |
+            set -x
+
+            fromIndex=$(params.fromIndex)
+            targetIndex=$(params.targetIndex)
+            binaryImage=$(params.binaryImage)
+
+            # New tag to replace with
+            newTag="$(tasks.get-ocp-version.results.stored-version)"
+
+            # Function to replace tag in an image
+            replace_tag() {
+                local image="$1"
```

Comment:
just remember to wrap variables inside `""` with `${}`

Reply:
Thanks for pointing that out. I almost missed that.

Reply:
It's not really needed, in most cases it will work even without curly brackets, but it's generally nicer/safer to use them.

And I think this thread is no longer relevant :)

Reply:
Fixed, thanks @mmalina  and @theflockers